scripts/lint

```diff
@@ -2,7 +2,6 @@
 
 cd "$(dirname "$0")/.."
 
-python3 -m black --check hass_nabucasa tests setup.py
-python3 -m flake8 hass_nabucasa setup.py tests
```

Comment:
flake8 does not support pyproject.toml, and it will be replaced by ruff shortly anyway, so it was removed for now.